core/includes/install.inc

```diff
@@ -920,31 +920,37 @@ function backdrop_verify_profile($install_state) {
   }
   $info = $install_state['profile_info'];
 
-  // Get a list of modules that exist in Backdrop's assorted subdirectories.
-  $present_modules = array();
+  // Get a list of projects that exist in Backdrop's assorted subdirectories.
+  $present_projects = array();
   foreach (backdrop_system_listing('/^' . BACKDROP_PHP_FUNCTION_PATTERN . '\.module$/', 'modules', 'name', 0) as $present_module) {
-    $present_modules[] = $present_module->name;
+    $present_projects[] = $present_module->name;
+  }
+  foreach (backdrop_system_listing('/^' . BACKDROP_PHP_FUNCTION_PATTERN . '\.info$/', 'themes', 'name', 0) as $present_theme) {
+    $present_projects[] = $present_theme->name;
+  }
+  foreach (backdrop_system_listing('/^' . BACKDROP_PHP_FUNCTION_PATTERN . '\.info$/', 'layouts', 'name', 0) as $present_layout) {
+    $present_projects[] = $present_layout->name;
   }
 
-  // The installation profile is also a module, which needs to be installed
+  // The installation profile is also a project, which needs to be installed
   // after all the other dependencies have been installed.
-  $present_modules[] = $profile;
+  $present_projects[] = $profile;
 
-  // Verify that all of the profile's required modules are present.
-  $missing_modules = array_diff($info['dependencies'], $present_modules);
+  // Verify that all of the profile's required projects are present.
+  $missing_projects = array_diff($info['dependencies'], $present_projects);
 
   $requirements = array();
 
-  if (count($missing_modules)) {
-    $modules = array();
-    foreach ($missing_modules as $module) {
-      $modules[] = '<span class="admin-missing">' . backdrop_ucfirst($module) . '</span>';
```

Comment:
We should not be capitalizing things here, as missing projects means no .info file, which means no pretty name - only machine name.